x-pack/plugins/ml/public/application/explorer/explorer_utils.js

```diff
@@ -427,12 +431,22 @@ export function loadAnnotationsTableData(selectedCells, selectedJobs, interval,
               return d;
             }),
           aggregations: resp.aggregations,
+          error: null,
         });
       })
       .catch((resp) => {
         console.log('Error loading list of annotations for jobs list:', resp);
-        // Silently fail and just return an empty array for annotations to not break the UI.
-        return resolve([]);
+        return resolve({
```

Comment:
I wonder if `extractErrorProperties` would also work for this to accommodate different error response shapes.

Reply:
Fixed in 83ffd4b.